packages/components/src/tree-grid/index.js

```diff
@@ -150,6 +150,9 @@ function TreeGrid(
 				}
 
 				// Focus the next element.
```

Comment:
Perhaps expand the comment here to clarify that it will not just focus the next element, but first expand the row if it is collapsed. Otherwise, the actions below contradict the statement in the comment.

Reply:
Made a better comment, looks good?

Reply:
Yes, thanks!